modular_skyrat/modules/modular_vending/code/autodrobe.dm

```diff
@@ -29,6 +29,7 @@
 				/obj/item/clothing/under/costume/skyrat/kamishimo = 5,
 				/obj/item/clothing/under/costume/skyrat/yukata = 5,
 				/obj/item/clothing/under/costume/skyrat/kimono = 5,
+				/obj/item/clothing/head/beret/badge = 5,
```

Comment:
Why the autodrobe and not just the clothesmate instead? It's not really a costume, it's a beret, should go with the beret in the clothesmate, I reckon.

Reply:
I swear that the beret is only in the autodrobe...

Reply:
I stand corrected.